snorkel/contrib/fonduer/fonduer/features/content_features.py

```diff
@@ -7,8 +9,8 @@
 from ..config import settings
 from ..lf_helpers import get_left_ngrams, get_right_ngrams, tokens_to_ngrams
 
-sys.path.append(os.path.join(os.environ['SNORKELHOME'], 'treedlib'))
-from treedlib import compile_relation_feature_generator
+#  sys.path.append(os.path.join(os.environ['SNORKELHOME'], 'treedlib'))
+#  from treedlib import compile_relation_feature_generator
```

Comment:
`treedlib` stuff is just commented out for now until we have treedlib that is python3 compatible.